airflow/api_connexion/openapi/v1.yaml

```diff
@@ -1572,6 +1572,196 @@ paths:
         '404':
           $ref: '#/components/responses/NotFound'
 
+  /auth-info:
+    get:
+      summary: Get site authentication information
+      x-openapi-router-controller: airflow.api_connexion.endpoints.auth_endpoint
+      operationId: get_auth_info
+      tags: [Authentication]
+      responses:
+        '200':
+          description: Success.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AuthInfo'
+
+  /auth/login:
+    post:
+      summary: Login user
+      x-openapi-router-controller: airflow.api_connexion.endpoints.auth_endpoint
+      operationId: auth_login
+      tags: [Authentication]
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/AuthDBLoginForm'
+      responses:
+        '200':
+          description: Success.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/JwtAuthAndUser'
+        '401':
+          $ref: '#/components/responses/Unauthenticated'
```

Comment:
This probably needs `400` too? Not sure about the rest of these.

Reply:
Since this is authentication endpoint, I'm making it return 401 whenever it fails